src/html_support_files/odoc.css

```diff
@@ -246,24 +246,42 @@ body {
 }
 
 body {
-  margin-left: calc(10vw + 20ex);
-  margin-right: 4ex;
-  margin-top: 20px;
-  margin-bottom: 50px;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0 4ex;
 }
 
 body.odoc {
-  max-width: 100ex;
+  max-width: 122ex;
+  display: grid;
+  grid-template-columns: 28ex 1fr;
+  column-gap: 4ex;
 }
 
 body.odoc-src {
   margin-right: calc(10vw + 20ex);
 }
 
+.odoc-content {
+  grid-row: 2;
+}
+
+.odoc-content > *:first-child {
+  /* This make the first thing in the content align with the sidebar */
+  padding-top: 0;
+  margin-top: 0;
+}
```

Comment:
The header (the first child) is not always small, and that makes strange result, I think.
See for instance the "odoc for authors" page of odoc's doc:

![image](https://github.com/ocaml/odoc/assets/34110029/5a4efe16-d791-4fa1-bd86-bd1bbd4f3bc8)

Why not, but I fear a much longer header would make it look strange, if the toc is pushed too much below...

Reply:
I am not sure I see what you mean

Reply:
Ah ! indeed. This is unrelated to that particular rule though. This can be changed by tweaking the grid layout (grid-row and grid-columns rules) (and if we do this rule becomes useless)
I agree it is a bit weird like that, but I think I still like it. I will ponder that a bit more.